source/org/zfin/marker/presentation/PreviousNameLight.java

```diff
@@ -154,4 +102,23 @@ public static boolean isUninformative(String name){
         if(name.contains(":") && false==transgenicPrefixPattern.matcher(name).matches()) return true ;
         return false ;
     }
+
+    public void setPureAliasName(String pureAliasName) {
```

Comment:
I would not overwrite these two setters but rather overwrite the getter
getAlias ()
and include the business logic of adding italicized strings.